Makefile

```diff
@@ -1,17 +1,11 @@
-up:
+up: build
```

Comment:
Why is `build` included here?

Reply:
I've included `build` so that `make up` will always launch the latest version of the code. It might be confusing if someone does a `git pull` and `make up` but the container does not get updated

Reply:
👌